docs/_future/getting_started_future.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Getting started
+order: 1
+slug: getting-started-future
+lang: en
+layout: tutorial
+permalink: /tutorial/getting-started-future
```

Comment:
Hi @hello-ashleyintech this seems to be a duplicate of [this](https://github.com/WilliamBergamin/bolt-python/pull/4#discussion_r1012134898) it was concluded it did not have a big impact on the behavior of the doc but If you feel more strongly towards it let me know I will update the path 💯

Reply:
sounds good! I agree that it doesn't really have a huge impact. Maybe to be consistent with Bolt JS you could also have the /future/getting-started stub for the site redirect to this tutorial in case folks try to navigate there based on Bolt JS?